tests/test_propertyMapPlot.py

```diff
@@ -0,0 +1,248 @@
+                names += [f"{coaddName}Coadd_{propertyName}_map_{operationName}"]
+
+        # Mock up corresponding HealSparseMaps and register them with the
+        # butler. Multiple maps allow us to check that we can generate multiple
+        # plots using a single tool.
+        mapsDict = {}
+        for name, value in zip(names, np.linspace(1, 10, len(names))):
+            hspMap = hsp.HealSparseMap.make_empty(nside_coverage=32, nside_sparse=4096, dtype=np.float32)
+            hspMap[0:10000] = value
+            hspMap[100000:110000] = value + 1
+            hspMap[500000:510000] = value + 2
+            datasetType = DatasetType(name, [], "HealSparseMap", universe=butler.dimensions)
+            butler.registry.registerDatasetType(datasetType)
+            dataRef = butler.put(hspMap, datasetType)
+            # Keys in mapsDict are designed to reflect the task's connection
+            # names, which are akin to datasettype names minus "_map".
+            mapsDict[name.replace("_map", "")] = DeferredDatasetHandle(
+                butler=butler, ref=dataRef, parameters=None
+            )
+
+        # Mock up the skymap and tractInfo.
+        skyMapConfig = DiscreteSkyMap.ConfigClass()
+        coords = [  # From the PS1 Medium-Deep fields.
+            (10.6750, 41.2667),  # M31
+            (36.2074, -04.5833),  # XMM-LSS
+        ]
+        skyMapConfig.raList = [c[0] for c in coords]
+        skyMapConfig.decList = [c[1] for c in coords]
+        skyMapConfig.radiusList = [2] * len(coords)
+        skyMapConfig.validate()
+        skymap = DiscreteSkyMap(config=skyMapConfig)
+        self.tractInfo = skymap.generateTract(0)
+
+        # Initialize the task and set class attributes for subsequent use.
+        task = PropertyMapTractAnalysisTask()
+        self.plotConfig = config
+        self.plotInfo = task.parsePlotInfo(mapsDict, dataId, list(mapsDict.keys()))
+        self.data = {"maps": mapsDict}
+        self.atool = PropertyMapTool()
+        self.atool.produce.plot.plotName = "test"
+        self.atool.finalize()
+
+    def tearDown(self):
+        removeTestTempDir(self.testDir)
+        del self.testDir
+        del self.atool
+        del self.data
+        del self.tractInfo
+        del self.plotConfig
+        del self.plotInfo
+
+    def test_PropertyMapTractAnalysisTask(self):
+        plt.rcParams.update(plt.rcParamsDefault)
+        result = self.atool(
+            data=self.data,
+            tractInfo=self.tractInfo,
+            plotConfig=self.plotConfig,
+            plotInfo=self.plotInfo,
+        )
+
+        # Previously computed reference hash values for the plots.
+        expectedHashList = [
+            "87235818c562587a6f6b17a233df76a1",
+            "90c68d7c6a70536485cea093b9401038",
+            "04385cbc6ead899d210fcaa628f88e45",
+            "b66ca03562bfcfcd758461c5ccb73837",
+            "98f145f450001642b2f1538c629f683f",
```

Comment:
I'd be very shocked if this works; and if it works today it certainly won't tomorrow.  Matplotlib images differ subtly from mac to linux and different os and matplotlib versions.  You can either use something like `test_scatterPlot.py` or check in sample images to the tests and use the matplotlib image comparison testing routines which allow some amount of variance from plotted image to reference image.